doc/user/content/ops/monitoring.md

```diff
@@ -175,3 +175,110 @@ Configuration parameter | Value
 `prometheus_url`        | `http://<materialized host>/metrics`
 `namespace`             | Your choice
 `metrics`               | `[mz*]` to select all metrics, or a list of specific metrics
+
+## Logging
+
+Materialize periodically emits messages to its [log file](/cli/#log-filter).
+These log messages serve several purposes:
+
+  * To alert operators to critical issues
+  * To record system status changes
+  * To provide developers with visibility into the system's execution when
+    troubleshooting issues
+
+As an operator, you should monitor for messages at the [`WARN` or `ERROR`
+levels](#levels). Every message at either of these levels indicates an
+issue that must be investigated and resolved.
+
+### Message format
+
+Each log message is emitted on its own line with the following format:
+
```

Comment:
In this context I would say "written to" rather than "emitted on", but you could also just say "Each log message is a single line in the following format:"